src/components/calculator/SavedDataSelector.tsx

```diff
@@ -48,19 +65,47 @@ export const SavedDataSelector: React.FunctionComponent<{
         <Typeahead
           clearButton={true}
           emptyLabel={t('calculator.no_prebuilt_scenario_found')}
-          multiple
+          filterBy={showMatchesAndCustomScenario}
           highlightOnlyResult={true}
           id="predefined-typeahead"
-          inputProps={{ autoComplete: 'chrome-off' }}
+          inputProps={{
+            autoComplete: 'chrome-off',
+            shouldSelectHint: (shouldSelect: boolean, event) => {
```

Comment:
I'm still confused as to what this does. In the currently releases version, highlighting an option with the arrow keys and pressing return selects it. Same in the deploy preview. Could you leave a comment to explain what this does?

Reply:
Start typing "Outdoor" and press tab.

Reset, start typing "Outdoor" and press enter. Nothing happens on the current website.

Reply:
added a comment. Better?